src/pages/editor/flow/components/EditorMinimap/index.tsx

```diff
@@ -0,0 +1,11 @@
+import { Card } from 'antd';
+import { Minimap } from 'gg-editor';
+import React from 'react';
+
+const EditorMinimap = () => (
+  <Card type="inner" size="small" title="Minimap" bordered={false}>
+    <Minimap height={200} />
+  </Card>
+);
+
+export default EditorMinimap;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of default exports is forbidden](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)